mmv1/products/compute/api.yaml

```diff
@@ -621,6 +621,31 @@ objects:
         name: 'cdnPolicy'
         description: 'Cloud CDN configuration for this Backend Bucket.'
         properties:
+        - !ruby/object:Api::Type::NestedObject
+          name: 'cacheKeyPolicy'
+          description: 'The CacheKeyPolicy for this CdnPolicy.'
+          properties:
+          - !ruby/object:Api::Type::Array
+            send_empty_value: true
```

Comment:
I think that not setting `send_empty_value` causes a problem that is similar to https://github.com/hashicorp/terraform-provider-google/issues/5273, but I'm not 100% sure about it because I am not quite familiar with Terraform.